internal/app/secretless/handlers/mysql/backend.go

```diff
@@ -1,13 +1,20 @@
 package mysql
 
 import (
+	"crypto/tls"
+	"errors"
 	"log"
 	"net"
 	"strconv"
 
 	"github.com/cyberark/secretless-broker/internal/app/secretless/handlers/mysql/protocol"
 )
 
+// Various errors the handler might return
+var (
+	ErrNoTLS = errors.New("SSL connection error: SSL is required but the server doesn't support it")
+)
+
 // ConfigureBackend resolves the backend connection settings and credentials and sets the
 // BackendConfig field.
 func (h *Handler) ConfigureBackend() (err error) {
```

Comment:
The `values` map defined above this func and used within it should be renamed to `connection_details` or `options`, since it's a map, and not the values of a map.